okio/src/commonTest/kotlin/okio/util.kt

```diff
@@ -0,0 +1,50 @@
+package okio
+
+import kotlin.random.Random
+
+fun Char.repeat(count: Int): String {
+  return toString().repeat(count)
+}
+
+fun segmentSizes(buffer: Buffer): List<Int> {
+  return generateSequence(buffer.head) { segment -> segment.next.takeIf { it !== buffer.head }}
```

Comment:
for tests this is fine, but for production code I prefer the obvious performance consequences of procedural code